### ISSUE TITLE
Accessibility Tools

### ISSUE THREAD

**Issue by FionaMacNeill** [mrbracchi/Awesome-ID#3]
**[W3C Teaching Materials Page](https://www.w3.org/WAI/curricula/)** 
Really helpful and adaptable materials for teaching accessibility 

**[WCAG Quick Reference ](https://www.w3.org/WAI/WCAG21/quickref/)** 
Essential and the filters on the left are really helpful when testing content for accessibility attributes. 

**[HeadingsMap browser add-on](https://rumoroso.bitbucket.io/)** 
Essential for any writing for the web to check your headings are in order in Blogs, SharePoint, most CMS's and also web-based learning object creation tools. 

**[WCAG Color Contrast Checker](https://rumoroso.bitbucket.io/)** 
Same author as HeadingsMap, super helpful for showing issues to stakeholders.

**[LanguageTool Grammar and Spell Checker](https://languagetool.org/)** 
Provides tips in any text editing box (WYSIWYG)

**[Landmark Navigation via Keyboard or Pop-up](http://matatk.agrip.org.uk/landmarks/)** 
Review the landmarks for keyboard-only navigation. 

**[Web Developer browser add-on for Mozilla Firefox](http://chrispederick.com/work/web-developer/firefox/)** 
Recommended by Web Accessibility Initiative (WAI). Particularly helpful for producing a quick report of images missing alternative text descriptions on a web page.
